transports/quartz/src/main/java/org/mule/transport/quartz/QuartzConnector.java

```diff
@@ -73,6 +76,11 @@ public QuartzConnector(MuleContext context)
         super(context);
     }
     
+    public String getFullName()
+    {
+    	return QUARTZ_SCHEDULER_PREFIX + muleContext.getConfiguration().getId() + "-" + getName();
```

Comment:
The original request to create a function was to avoid the code duplication which is still existent. If you add parameters to pass the congif ID and the name and make it static, then you can use everywhere